.github/workflows/main.yml

```diff
@@ -162,13 +162,13 @@ jobs:
 
   imaging-api-tests:
     runs-on: ubuntu-22.04
-    timeout-minutes: 10
+    # timeout-minutes: 10
```

Comment:
Note to self so I don't forget, enable this again when the container has rebuilt